general-guidelines/validating-api-descriptions.md

```diff
@@ -0,0 +1,74 @@
+# Validating API Description
+
+API description files (e.g. OpenAPI specification for REST APIs) SHOULD be validated to verify whether they comply with this guidelines. In order to do that you can utilize [Spectral](https://github.com/stoplightio/spectral) ruleset implementing them.

```

Comment:
I'd rephrase it the other way around – we are not doing governance here – we are helping others to follow the guidelines.

Something like this: 

> If you want to follow adidas API design guidelines for REST APIs you MAY validate your OAS file using ....

Reply:
changed